CHANGELOG.md

```diff
@@ -10,6 +10,33 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Fixed
 
+## [0.39.1](https://github.com/returntocorp/semgrep/releases/tag/v0.39.1) - 2021-01-26
+
+No new changes in this version.
+This is a re-release of 0.39.0 due to an error in the release process.
+
+## [0.39.0](https://github.com/returntocorp/semgrep/releases/tag/v0.39.0) - 2021-01-26
+
+### Added
+
+- Support rules in JSON or JSonnet format (#2428).
+- Support new nested format for combining patterns into a boolean
```

Comment:
Same for this one. It's not available in "regular" semgrep.

Reply:
Thanks! Could you review https://github.com/returntocorp/semgrep/pull/2469/commits/d6975cfebbd6a623bc88b42a6b04b8b912a6058c and approve if the new changelog looks okay?